.github/workflows/publish-helm-chart.yml

```diff
@@ -0,0 +1,90 @@
+name: Publish Helm Chart
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
We also want to run this on pull requests! In particular, we want to make sure that the helm checks out (`helm lint`) and builds `helm build`

Reply:
Yeah, I think it'd be good to run `helm lint` in a step before we attempt the `helm package` (I thought it was `helm build` but I was wrong!)